RESOURCES/INTHEWILD.md

```diff
@@ -160,6 +160,9 @@ Join our growing community!
 ### Government
 - [City of Ann Arbor, MI](https://www.a2gov.org/) [@sfirke]
 
+### Travel
+- [Skyscanner](https://www.skyscanner.net/)
```

Comment:
@cleslie you may also consider adding your GitHub handle after the entry (see line 161 above)

Reply:
Have added, thanks !